jps/jupedsim/IO/worldparser.py

```diff
@@ -0,0 +1,137 @@
+import re
+from typing import List
+
+import ezdxf
+import jpscore
```

Comment:
I would suggest only to import geometry sub-module
```suggestion
from jpscore import geometry
```